selfapi.js

```diff
@@ -405,21 +430,30 @@ API.prototype = {
       html += '<h2 id="' + getAnchor(title) + '">' + title + '</h2>\n';
       html += '<pre>' + method.toUpperCase() + ' ' + fullPath + '</pre>\n';
       if (handler.description) {
-        html += '<p>' + handler.description + '</p>\n';
+        html += '<p>' + handler.description.replace(/\n/g, '<br/>') + '</p>\n';
       }
 
       if (handler.examples && handler.examples.length > 0) {
         var example = handler.examples[0];
 
         var request = example.request || {};
-        if (request.headers || request.body) {
+        if (request.headers || request.body || request.urlParameters) {
           html += '<h3>Input</h3>\n<pre>';
+
+          // Format example-specific URL
+          var exampleURL = method.toUpperCase() + ' ' + fullPath;
+          var requestParameters = request.urlParameters || {};
+          for (var requestParam in requestParameters) {
```

Comment:
Nit: Please try to use full words in variable names (local style... sorry about that). E.g. here you could use `for (var parameter in requestParameters)`.